datahub-graphql-core/src/main/resources/auth.graphql

```diff
@@ -11,6 +11,12 @@ extend type Query {
     List access tokens stored in DataHub.
     """
     listAccessTokens(input: ListAccessTokenInput!): ListAccessTokenResult!
+
+    """
+    Fetches the metadata of an access token.
+    This is useful to debug when you have a raw token but don't know the actor.
+    """
+    getAccessTokenMetadata(token: String!): AccessTokenMetadata!
```

Comment:
Can you add this endpoint to our docs? It's in `personal-access-tokens.md` doc file

Reply:
Pretty sure GraphQL endpoints get docs auto generated

Reply:
yes that is what I was thinking too. Graphiql interface shows the docs.

Reply:
verified it is auto-generated in graphiql 
![image](https://github.com/datahub-project/datahub/assets/4127841/4cbf2af6-f529-43ce-ba5e-2cf1e86cb6f8)

Reply:
Yes GraphQL does have auto documentation. The file I linked to is a user guide. Basically put in the docs somewhere that this endpoint also exists and why it is useful.

Reply:
These also already show up in the docs website: https://datahubproject.io/docs/graphql/queries